lib/commands/startTest.js

```diff
@@ -45,5 +42,14 @@ async function startTest(clientTestId, {name, description} = {}) {
 	testContext.setContext(content);
 	logger.info(testWasStarted(name || clientTestId));
 }
+const deprecateStartTest = util.deprecate(startTest, startTestDeprecate());
+
+function startTestWrapper(...args) {
```

Comment:
You can check `arguments` object for displaying deprecation warning inside `startTest`

Reply:
That's native way for it https://nodejs.org/dist/latest-v14.x/docs/api/util.html#util_util_deprecate_fn_msg_code and as with other places probably it should be shown just once but not every time when used